EsLint dependency version mismatch in atlas components

We currently have a problem building the UI packages.

From the above log I can see these errors that is related to a version mismatch of `eslint` library.

This is causing issues in these components:
- scxa-cell-type-wheel-experiment
- experiments-summary-panel
- species-summary-panel

```
All dependencies match the latest package versions :)
>> /Users/karoly/dev/gxa/atlas-web-single-cell/app/src/main/javascript/bundles/cell-type-wheel-heatmap$ rm -rf node_modules package-lock.json
>> /Users/karoly/dev/gxa/atlas-web-single-cell/app/src/main/javascript/bundles/cell-type-wheel-heatmap$ npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: @ebi-gene-expression-group/scxa-cell-type-wheel-heatmap@1.0.0
npm ERR! Found: eslint@8.57.1
npm ERR! node_modules/eslint
npm ERR!   dev eslint@"^8.17.0" from the root project
npm ERR!   peer eslint@">= 8" from @ebi-gene-expression-group/eslint-config@3.5.1
npm ERR!   node_modules/@ebi-gene-expression-group/eslint-config
npm ERR!     dev @ebi-gene-expression-group/eslint-config@"^3.5.1" from the root project
npm ERR!   4 more (eslint-config-standard, eslint-plugin-import, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer eslint@">=9" from eslint-plugin-cypress@4.0.0
npm ERR! node_modules/eslint-plugin-cypress
npm ERR!   peer eslint-plugin-cypress@">= 2" from @ebi-gene-expression-group/eslint-config@3.5.1
npm ERR!   node_modules/@ebi-gene-expression-group/eslint-config
npm ERR!     dev @ebi-gene-expression-group/eslint-config@"^3.5.1" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/karoly/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/karoly/.npm/_logs/2024-10-15T13_16_23_323Z-debug-0.log
>> /Users/karoly/dev/gxa/atlas-web-single-cell/app/src/main/javascript/bundles/cell-type-wheel-heatmap$ npm audit fix
npm ERR! code ENOLOCK
npm ERR! audit This command requires an existing lockfile.
npm ERR! audit Try creating one first with: npm i --package-lock-only
npm ERR! audit Original error: loadVirtual requires existing shrinkwrap file

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/karoly/.npm/_logs/2024-10-15T13_16_24_493Z-debug-0.log
```

--- a/app/src/main/javascript/bundles/cell-type-wheel-heatmap/package.json
+++ b/app/src/main/javascript/bundles/cell-type-wheel-heatmap/package.json
@@ -16,9 +16,5 @@
     "@ebi-gene-expression-group/scxa-cell-type-wheel-heatmap": "^1.4.14",
     "react": "^17.0.2",
     "react-dom": "^17.0.2"
-  },
-  "devDependencies": {
-    "@ebi-gene-expression-group/eslint-config": "^3.5.1",
-    "eslint": "^8.17.0"
   }
 }

--- a/compile-front-end-packages.sh
+++ b/compile-front-end-packages.sh
@@ -66,7 +66,6 @@ for MODULE_DIR in `ls`
 do
   pushd .
   cd $MODULE_DIR
-  update_npm_package
   echo ">> $PWD$ npm run prepare"
   npm run prepare
   popd

